### DISCUSSION TITLE
Yetiforce changelog - wrong URL in Readme.md

### DISCUSSION THREAD

**Discussion by apcloic** [YetiForceCompany/YetiForceCRM#14999] [General] [answered]
Hi,

While I was searching for Yeti 6.2 changelog, I found that the URL provided in [README.md](https://github.com/YetiForceCompany/YetiForceCRM/blob/developer/README.md) was incorrect
It should be replaced by :
https://yetiforce.com/en/knowledge-base/documentation/developer-documentation/category/changes-2
Regards

**Comment by rskrzypczak** [comment, marked as answer]
Fixed.  Thanks.
